tests/gfmdataprocessor/blockquotes.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { testDataProcessor as test } from '/tests/markdown-gfm/_utils/utils.js';
+
+describe( 'GFMDataProcessor', () => {
+	describe( 'blockquotes', () => {
+		it( 'should process single blockquotes', () => {
+			test(
+				'> foo bar',
+
+				// GitHub is rendering as:
+				//
+				// <blockquote>
+				// <p>foo bar</p>
+				// </blockquote>
+				'<blockquote><p>foo bar</p></blockquote>'
+			);
+		} );
+
+		it( 'should process nested blockquotes', () => {
+			test(
+				'> foo\n' +
```

Comment:
I think that for readability, we should rather use template strings. Outdent them so they start at xpos=0 and that's it.